src/components/data-grid/header-column-toggle/HeaderColumnToggle.tsx

```diff
@@ -73,11 +98,22 @@ export const HeaderColumnToggle = ({ api: gridApi, columnApi }: HeaderColumnTogg
           optionProps: () => ({ [DATA_TEST_ID]: 'data-grid-column-toggle-option' }),
         }}
       >
-        <TertiaryButton>
-          <TableSettings size={scale600} color={active ? primary : '#87878E'} />
+        <TertiaryButton
+          overrides={{
+            BaseButton: {
+              style: {
+                padding: '0px',
```

Comment:
It's important to always use the helpers. Otherwise you get potential style bugs

```suggestion
                ...padding('0'),
```